src/components/breadcrumbs/BreadcrumbBackLink.tsx

```diff
@@ -0,0 +1,22 @@
+import React, { FC } from 'react';
+import { chakra, useMultiStyleConfig } from '@chakra-ui/react';
+
+import Link from '../link';
+import { ArrowLeftIcon as BackArrowIcon } from '../icons';
+
+const BreadcrumbBackLink: FC = ({ ...props }) => {
```

Comment:
I don't see a benefit in this component. We have the Link component fulfilling the need for the arrow link already. 
Regarding the separator, according to the PDP design, it's not visible on mobile so it might be more straight forward when we hide it in the project.